randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -1432,12 +1440,14 @@ Extra - room_id: [40]
               # Vanilla Method - Shinespark from Hornoad Housing -> Access
               Speed Booster and Disabled Door Lock Rando and Disabled Entrance Rando and Can Kill Eye Door
               Any of the following:
-                  # Pre-Fight, with Hi-Jump
+                  # Pre-Fight, with Hi-Jump: https://youtu.be/q7HKdMt4v3g
                   Hi-Jump and Shinespark Tricks (Intermediate)
-                  # After Fight, Trickless
+                  # Pre-Fight, No Hi-Jump: https://youtu.be/Fo8-jjuPdik
+                  Shinespark Tricks (Intermediate)
```

Comment:
this doesnt make sense.
if pre-NHJ is on a lower or equal trick level than post-NHJ, then logic will always choose this path and the `After Core X` requirement becomes redudant.
This either needs to be expert (which doesnt feel right to me), or we axe the boss requirement.

Reply:
The trick for each 4 of the variety is pretty much the same imo. The only difference I feel is that doing this before the boss fight has the slight chance of softlock if no progression is in the etank location. The only difficulty changes when your only way to kill the hornoads is an without beam upgrades, which is possible, just annoying

Reply:
For now, I'll match the other combat difficulties so they should be equally likely, but if you have another way you would want this handled I'm more than willing to make further adjustments.

Reply:
if the trick for all of them is the same then they're not worth listing. they all get useless and logic will always choose prefight NHJ.
Please do what i said in the first comment of this thread: either adjust the shinespark requirements for all to be unique, or axe the boss requirements for ones where you feel are too similar.